### DISCUSSION TITLE
SVG loader

### DISCUSSION THREAD

**Discussion by realprabs** [callstack/repack#58] [Q&A] [answered]
Do you know any webpack loader for `react-native-svg`? I was using https://github.com/kristerkari/react-native-svg-transformer with metro. Tried https://www.npmjs.com/package/react-native-svg-loader, but it didn't work. It was published 2 years back, looks like not maintained.

I tried https://www.npmjs.com/package/@svgr/webpack with `native: true`. That just throws `NonErrorEmittedError: (Emitted value instead of an instance of Error) Error in parsing SVG: Non-whitespace before first tag.` for all SVGs.

**Comment by zamotany** [comment, marked as answer]
I have opened a PR to make it possible to render SVGs with updated documentation and dedicated SVG guide. You might want to track this issue: #73.
